astrowidgets/tests/test_api_ginga.py

```diff
@@ -10,7 +10,13 @@
 
 from ginga.ColorDist import ColorDistBase
 
-from ..core import ImageWidget, ALLOWED_CURSOR_LOCATIONS
+from astrowidgets.ginga import ImageWidget, ALLOWED_CURSOR_LOCATIONS
+from astrowidgets.interface_definition import ImageViewerInterface
+
+
+def test_consistent_interface():
+    iw = ImageWidget()
+    assert isinstance(iw, ImageViewerInterface)
```

Comment:
Does this actually do the test of whether the API matches with no code changes needed in the ginga object at all?  If so, that's very magical! (in a good way)

Reply:
Yes -- it does not check signatures, etc, but it makes sure that all of the properties/methods are there.